src/components/Property/PropertyDetails.js

```diff
@@ -2,52 +2,81 @@ import PropTypes from 'prop-types'
 import LocationAlerts from './LocationAlerts'
 import PersonAlerts from './PersonAlerts'
 import Tenure from './Tenure'
+import Link from 'next/link'
 
 const PropertyDetails = ({
   address,
   hierarchyType,
   locationAlerts,
   personAlerts,
   tenure,
-}) => (
-  <div>
-    <h1 className="govuk-heading-l">
-      {hierarchyType.subTypeDescription}: {address.addressLine}
-    </h1>
-
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-one-half">
-        <div className="govuk-body-s">
-          <div className="property-details-main-section">
-            <span className="govuk-body-xs">Property details</span>
-            <br></br>
-            <span className="govuk-!-font-weight-bold text-green">
-              {address.addressLine}
-            </span>
-            <br></br>
-            {address.streetSuffix && (
-              <>
-                <span className="govuk-!-font-weight-bold text-green">
-                  {address.streetSuffix}
-                </span>
-                <br></br>
-              </>
-            )}
-            <span className="govuk-body-xs text-green">
-              {address.postalCode}
-            </span>
+}) => {
+  let repairsStatus = (status, subTypeDescription) => {
+    if (status) {
+      return (
+        <span class="govuk-heading-m">
+          <strong>
+            <Link href="">
+              <a>Raise a repair on this {subTypeDescription.toLowerCase()} </a>
+            </Link>
+          </strong>
+        </span>
+      )
+    } else {
+      return (
+        <div class="govuk-warning-text">
+          <span class="govuk-warning-text__icon" aria-hidden="true">
+            !
+          </span>
+          <strong class="govuk-warning-text__text">
+            <span class="govuk-warning-text__assistive">Warning</span>
+            Cannot raise a repair on this property due to tenancy status
+          </strong>
+        </div>
+      )
+    }
+  }
+  return (
+    <div>
+      <h1 className="govuk-heading-l">
+        {hierarchyType.subTypeDescription}: {address.addressLine}
+      </h1>
+      <div>
+        {repairsStatus(tenure.canRaiseRepair, hierarchyType.subTypeDescription)}
```

Comment:
I like our `canRaiseRepair`  property name and I feel like it's almost asking us to be used in an `if` condition directly for a bit of a readability boost, rather than passed down into our helper-type function here.

With a couple of smaller functions, I wonder if this approach might be a bit more readable:

```
{
  if(tenure.canRaiseRepair) {
    renderRaiseRepairLink(hiearchyType.subTypeDescription)
  } else {
    renderRepairUnavailableText()
  }
}
```

Reply:
I'd agree with Neil and also perhaps extract this function into its own component. For readability, it helps that the file doesn't get too big and then you can pass down the state of whatever you need as props